src/SonarQube.TeamBuild.Integration.XamlBuild/TfsLegacyCoverageReportProcessor.cs

```diff
@@ -90,6 +90,13 @@ public TfsLegacyCoverageReportProcessor()
             return continueProcessing;
         }
 
+        protected override bool TryGetTrxFile(AnalysisConfig config, ITeamBuildSettings settings, ILogger logger, out string trxFilePath)
+        {
+            // We don't support TFS Legacy builds for automatic TRX discovery
```

Comment:
Pleaes explain why.

Reply:
There is no ticket to support that

Reply:
I meant in the comment for future ourselves :)

Xaml is a legacy feature, so fine with me not to bother.